ycms/cms/templates/_base.html

```diff
@@ -95,21 +105,31 @@
             </ul>
             <div class="absolute bottom-0 w-full text-center text-gray-600">
                 <li class="mb-4" id="timetravel-start">
-                    <a href="?time=2024-01-01T00%3A00%3A00.000Z" class="justify-center"><i icon-name="timer-reset"></i> {% translate "Start Timetravel" %}</a>
-                </li>
-                <li class="hidden mb-4" id="timetravel-stop">
-                    <a href="?" class="justify-center"><i icon-name="timer-off"></i> {% translate "Stop Timetravel" %}</a>
-                </li>
-                <p class="text-sm">
-                    {% translate "Logged in as" %} <span class="text-blue-500">{{ user.first_name }} {{ user.last_name }}</span>
-                </p>
-                <p class="tracking-wide">© YCMS 2023</p>
-            </div>
-        </div>
-        <div class="flex-1 p-6 ml-0 lg:ml-64 h-full overflow-y-auto">
-            {% block content %}
-            {% endblock content %}
-            {% include "timetravel_controls.html" %}
-        </div>
+                <a href="?time=2024-01-01T00%3A00%3A00.000Z" class="justify-center"><i icon-name="timer-reset"></i>
+            {% translate "Start Timetravel" %}</a>
+        </li>
+        <li class="hidden mb-4" id="timetravel-stop">
+            <a href="?" class="justify-center"><i icon-name="timer-off"></i> {% translate "Stop Timetravel" %}</a>
+        </li>
+        <button id="theme-toggle"
+                type="button"
+                class="text-gray-500 dark:text-gray-400 hover:bg-gray-100 dark:hover:bg-gray-600 focus:outline-none focus:ring-4 focus:ring-gray-200 dark:focus:ring-gray-800 rounded-lg text-sm p-2.5">
```

Comment:
```suggestion
                class="text-gray-500 dark:text-gray-400 hover:bg-gray-100 dark:hover:bg-gray-600 focus:outline-none focus:ring-4 focus:ring-gray-200 dark:focus:ring-gray-800 rounded-lg text-sm p-2.5 mb-4">
```
Optional.

Also optional, but maybe adding "Switch Theme" or something like that could make it clearer to users? I mean, we know what the button does, but not everyone might.

Reply:
I add a tooltip to display its purpose when users hover over it.